packages/babel-preset-gatsby-package/index.js

```diff
@@ -30,7 +39,8 @@ function preset(context, options = {}) {
             loose: true,
             debug: !!debug,
             shippedProposals: true,
-            modules: `commonjs`,
+            modules: esm ? false : `commonjs`,
+            bugfixes: esm,
```

Comment:
For anyone interested what this option is: https://babeljs.io/blog/2020/03/16/7.9.0#babelpreset-envs-bugfixes-option-11083httpsgithubcombabelbabelpull11083